nvme.c

```diff
@@ -6573,14 +6575,18 @@ static int copy_cmd(int argc, char **argv, struct command *cmd, struct plugin *p
 	const char *d_format = "source range entry format";
 
 	_cleanup_nvme_dev_ struct nvme_dev *dev = NULL;
-	uint16_t nr, nb, ns, nrts, natms, nats;
+	uint16_t nr, nb, ns, nrts, natms, nats, nid;
 	__u16 nlbs[128] = { 0 };
 	__u64 slbas[128] = { 0 };
+	__u32 snsids[128] = { 0 };
+	__u16 sopts[128] = { 0 };
 	int err;
 
 	union {
 		__u32 f0[128];
 		__u64 f1[101];
```

Comment:
Pre-existing, but where does the 101 range limit come from?

Reply:
Not sure. I think it should be ok to be 128 for all of them.

Reply:
Nevermind, I think it's in the spec that 1 and 3 should have 102 max? In that case though, it should be 102, not 101.

Reply:
I don't see anything in the spec saying that there can only be 128 or 102 entries. It looks like that's how many there are in the examples, but up to 256 ranges should be allowed given that NR is an 8-bit 0-based value?